packages/cfd/src/Constants/cfd-password-modal-constants.ts

```diff
@@ -0,0 +1,31 @@
+import { localize } from '@deriv/translations';
```

Comment:
@aizad-deriv I'm not sure but based on this [thread](https://deriv-group.slack.com/archives/C3AHMSM3P/p1692768585311499) we have to use `Localize` instead of `localize`. You used this localize function in many places of this PR.